broker-cli/commands/wallet.js

```diff
@@ -367,9 +369,10 @@ module.exports = (program) => {
     .command('wallet balance', 'Current daemon wallet balance')
     .command('wallet new-deposit-address', 'Generates a new wallet address for a daemon instance')
     .argument('<symbol>', `Supported currencies for the exchange: ${SUPPORTED_SYMBOLS.join('/')}`)
-    .command('wallet commit-balance')
+    .command('wallet commit')
     .argument('<symbol>', `Supported currencies for the exchange: ${SUPPORTED_SYMBOLS.join('/')}`)
     .argument('[amount]', 'Amount of currency to commit to the relayer', validations.isDecimal)
+    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
```

Comment:
Might be good to have available market names in the description

Reply:
Will do that here: https://trello.com/c/a5xtnFbv/655-get-supported-markets-ccxt